libs/libslirp/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libslirp
+PKG_VERSION:=4.6.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.freedesktop.org/slirp/$(PKG_NAME)/-/archive/v$(PKG_VERSION)
```

Comment:
Use codeload.

Reply:
how? Isn't codeload a service on GitHub when this source is at Freedesktop's...

Reply:
Ah I misread.